src/gui/auth/qgsauthconfigeditor.h

```diff
@@ -48,6 +48,9 @@ class GUI_EXPORT QgsAuthConfigEditor : public QWidget, private Ui::QgsAuthConfig
     //! Hide the widget's title, e.g. when embedding
     void toggleTitleVisibility( bool visible );
 
+    //! Returns the list of selected authentication configuration IDs
```

Comment:
since 3.20 ? 3.22?

Reply:
3.20 is my preference

Reply:
Noticed on the other methods, just add the api comment here also.

It's ok for me, code looks good. If @elpaso doesn't put a veto (and it looked to me like he doesn't), I think that can be done.

Reply:
@m-kuhn no vetos from my side, I just think that JSON would be better and could reuse the current implementation for oauth.

Reply:
@elpaso , the reasons to go XML are 1/ QGIS is mostly writing XML format (style export, metadata export, bookmarks export, not to mention project files :)), and it'll make it easier to embed into project files later. I also thought adding the writeXml and readXml API matches established functions we have elsewhere (symbology, layout, etc)